stats/gRPC.md

```diff
@@ -0,0 +1,125 @@
+| grpc.io/client/uncompressed_request_bytes  | By   | Total uncompressed bytes sent in a gRPC request.                                              |
+| grpc.io/client/response_count              |      | Number of response messages in a gRPC request.                                                |
+| grpc.io/client/response_bytes              | By   | Total compressed bytes received in a gRPC request.                                            |
+| grpc.io/client/uncompressed_response_bytes | By   | Total uncompressed bytes received in a gRPC request.                                          |
+| grpc.io/client/latency                     | ms   | Time between first byte of request received to last byte of response sent, or terminal error. |
+| grpc.io/client/server_latency              | ms   | Propagated from the server and should have the same value as "grpc.io/server/latency".        |
+
+### Tags
+
+All client metrics should be tagged with the following. Some tags are not available for measurements recorded at the beginning of the gRPC request (e.g. `grpc.canonical_status`).
+
+| Tag name              | Description                                                   |
+|-----------------------|---------------------------------------------------------------|
+| grpc.client_method    | Full gRPC method name, including package, service and method. |
+| grpc.canonical_status | gRPC canonical status code.                                   |
+
+### Default views
+
+The following set of views are considered minimum required to monitor client side performance:
+
+| View name                                   | Measure suffix              | Aggregation  | Tags suffix                     |
+|---------------------------------------------|-----------------------------|--------------|---------------------------------|
+| grpc.io/client/started_count                | started_count               | count        | client_method                   |
+| grpc.io/client/request_bytes                | request_bytes               | distribution | client_method                   |
+| grpc.io/client/response_bytes               | response_bytes              | distribution | client_method                   |
+| grpc.io/client/latency                      | latency                     | distribution | client_method                   |
+| grpc.io/client/finished_count               | latency                     | count        | client_method, canonical_status |
+
+### Extra views
+
+The following set of views are considered useful but not mandatory to monitor client side performance:
+
+| View name                                   | Measure suffix              | Aggregation  | Tags suffix                     |
+|---------------------------------------------|-----------------------------|--------------|---------------------------------|
+| grpc.io/client/request_count                | request_count               | distribution | client_method                   |
```

Comment:
isn't this "started_count"?

Reply:
the problem is "request" here means "number of request messages" in a streaming request, right?

Reply:
Changed the name to request_messages_count for the old request_count.